src/goto-analyzer/goto_analyzer_parse_options.cpp

```diff
@@ -252,6 +254,20 @@ void goto_analyzer_parse_optionst::get_command_line_options(optionst &options)
     options.set_option("verify", true);
     options.set_option("general-analysis", true);
   }
+  else if(cmdline.isset("get-function-pointer-values"))
+  {
+    options.set_option(
+      "get-function-pointer-values",
+      cmdline.get_value("get-function-pointer-values"));
+    options.set_option("variable-sensitivity", true);
+    options.set_option("domain set", true);
+    options.set_option("pointers", true);
+    options.set_option("arrays", true);
+    options.set_option("structs", true);
+    options.set_option("value-set", true);
+    options.set_option("verify", true);
+    options.set_option("general-analysis", true);
```

Comment:
Some why comments here would be good (in particular, why verify and general analysis need to be set to true)